srcpkgs/gtkcord3/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'gtkcord3'
+pkgname=gtkcord3
+version=0.0.4
+revision=1
+build_style=go
+go_import_path=github.com/diamondburned/gtkcord3/
+makedepends="libhandy-devel gtk+3-devel pkg-config"
+depends="libhandy gtk+3"
```

Comment:
Don't need this here

Reply:
I unfortunately didn't see this until after I forced pushed.